src/SmartComponents/DriftPage/DriftTable/DriftTable.js

```diff
@@ -38,13 +40,31 @@ export class DriftTable extends Component {
         await window.insights.chrome.auth.getUser();
         const { fetchCompare } = this.props;
 
-        fetchCompare(this.systemIds, this.baselineIds, this.HSPIds);
+        fetchCompare(this.systemIds, this.baselineIds, this.HSPIds, this.referenceId);
+    }
+
+    moveSystem = (systems) => {
```

Comment:
We are generally running into this problem, where we are dealing with baselines, systems and historical profiles. So the use of system nomenclature can get confusing. Naming ideas then `shiftReferenceToFront` and pass in `items` or `masterList`